modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/TypeScriptFetchClientCodegen.java

```diff
@@ -117,6 +123,16 @@ public String getHelp() {
         return "Generates a TypeScript client library using Fetch API (beta).";
     }
 
+    @Override
+    public String apiDocFileFolder() {
+        return (outputFolder + File.separator + apiDocPath);
+    }
+
+    @Override
+    public String modelDocFileFolder() {
+        return (outputFolder + File.separator + modelDocPath);
```

Comment:
can you please add `.replace('/', File.separatorChar)` ?

output folder may contains `/`

Reply:
I believe I've updated it to what you wanted

Reply:
the build failed. please take a look when you've time.

Reply:
@wing328 Not sure what the issue was but have pushed changes to run and test the build again